src/preProcessing/preProcessUnsupportedParserFeatures.ts

```diff
@@ -0,0 +1,24 @@
+import * as Babel from '@babel/types';
+import { PreparedTemplate } from './preProcessingTypes';
+import { replaceBlockStatementsWithinAttributes, replaceBlockStatementsAroundAttributes } from './blockStatementsWithinNodeTag';
+import replaceAttributeGeneratingHelpers from './replaceAttributeGeneratingHelpers';
+
+const preProcessSteps = [
+  replaceBlockStatementsWithinAttributes,
+  replaceBlockStatementsAroundAttributes,
+  replaceAttributeGeneratingHelpers, // NOTE: this preprocessor has a dependency on having already converted attributes that contain helpers within or around their value
+];
+
+const preProcessUnsupportedParserFeatures = (handlebarsTemplate: string):PreparedTemplate => {
```

Comment:
TODO: look into proper parser support for these features, to avoid the need for flaky pre-processing